examples/vec/src/main.sw

```diff
@@ -25,8 +25,11 @@ fn main() {
     // ANCHOR_END: vec_get_oob
     // ANCHOR: vec_iterate
     let mut i = 0;
-    while i < v.len() {
-        log(v.get(i).unwrap());
+    while i < v
+        .len() {
+        log(v
+            .get(i)
+            .unwrap());
```

Comment:
This one also seems a bit weird, how it was previously looks correct to me

Reply:
I'm on it to fix it.

Reply:
Does it look better now @eureka-cpu ?

Reply:
They do look better, I think there's a few odd ones here and there but overall it's a big improvement. I'll defer approval to @JoshuaBatty